src/Commands/Draw2Command.h

```diff
@@ -14,6 +14,6 @@ class Draw2Command : public Command {
 
 	void Execute() const override {
 		std::cout << "[Command] Draw 2 cards." << std::endl;
-		this->_executor->ForceDrawNextPhase(2);
+		this->_executor->ForceDrawNextPhase(2, true);
```

Comment:
suggestion: use const to explain what 2 or 4 means in the code instead using magic numbers